schema/models.py

```diff
@@ -161,19 +148,23 @@ class Meta:
 
 class MemberGroup(models.Model):
 	id = models.AutoField(primary_key=True)
-	member = models.ForeignKey(settings.AUTH_USER_MODEL)
-	group = models.ForeignKey('Group')
+	member = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+	group = models.ForeignKey('Group', on_delete=models.CASCADE)
 	timestamp = models.DateTimeField(auto_now=True)
 	admin = models.BooleanField(default=False)
-	moderator = models.BooleanField(default=False)
+	all_emails = models.BooleanField(default=True)
```

Comment:
There are many attributes for this table. Would it be possible to add comments for each attribute to describe what they mean?

Reply:
I will comment and push changes with comments to schema.